aequilibrae/project/field_editor.py

```diff
@@ -105,7 +105,7 @@ def _check_completeness(self) -> None:
             for field in raw_fields:
                 if field[-3:] == '_ab':
                     if field[:-3] + '_ba' in raw_fields:
-                        fields.append(field[:-2])
+                        fields.append(field[:-3])
```

Comment:
I know this is just a bug fix... for this block but....


```
fields = [re.sub("_ab", "", re.sub("_ba", '', f)) for f in fields].unique
```

Reply:
Like this?

Reply:
Great suggestion. Incorporated with a syntax correction. Thanks!